codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -140,6 +143,100 @@ public void onPriorityTreeParentChanging(Http2Stream stream, Http2Stream newPare
                 }
             }
         });
+
+        if (listener == null) {
```

Comment:
Lets re-asses after I push new code. There is only 1 null check which exists elsewhere to prevent a NPE.  This null check implicitly guards against potential of listener NPE elsewhere.